db/migrate/20151117005827_create_review_chats.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReviewChats < ActiveRecord::Migration
+  def change
+    create_table :review_chats do |t|
+      t.integer :assignment_id
```

Comment:
We have performed this migration and pushed the code to our git. We are making the other changes that you suggested us in the meanwhile